packages/grafana-data/src/dataframe/__snapshots__/ArrowDataFrame.test.ts.snap

```diff
@@ -102,3 +102,171 @@ Array [
   },
 ]
 `;
+
+exports[`Read/Write arrow Table to DataFrame should parse output with dataframe 1`] = `
+Object {
+  "fields": Array [
+    Object {
+      "config": Object {},
+      "labels": undefined,
+      "name": "string_values",
+      "type": "string",
+      "values": Array [
+        "Grafana",
+        "❤️",
+        "Transforms",
+      ],
+    },
+    Object {
+      "config": Object {
+        "decimals": 2,
+        "filterable": false,
+        "links": Array [
+          Object {
+            "targetBlank": true,
+            "title": "Donate - The Sloth Conservation Foundation",
+            "url": "https://slothconservation.com/how-to-help/donate/",
+          },
+        ],
+        "max": null,
+        "min": null,
+        "noValue": "😤",
+        "nullValueMode": "null",
+        "title": "Grafana ❤️ (Previous should be heart emoji) 🦥 (Previous should be sloth emoji)",
+      },
+      "labels": undefined,
+      "name": "nullable_string_values",
+      "type": "string",
+      "values": Array [
+        "🦥",
+        null,
+        "update your unicode/font if no sloth, is 2019.",
+      ],
+    },
+    Object {
+      "config": Object {
+        "max": 1,
+        "min": 0,
+      },
+      "labels": undefined,
+      "name": "int_values",
+      "type": "number",
+      "values": Array [
+        "\\"-9223372036854775808\\"",
+        "\\"1\\"",
+        "\\"9223372036854775807\\"",
```

Comment:
something weird here... I will investigate if this is just the toJSON or a real issue